### PR TITLE
Code documentation

### PR DESCRIPTION
**Pull request by davegomez**
This PR addressed issue #4

### CODE REVIEW

File: dist/aw-grid.css [hunk 1]
```diff
@@ -550,8 +550,8 @@
         -webkit-box-flex: 0;
         -ms-flex: 0 0 auto;
             flex: 0 0 auto;
-        padding-left: .5rem;
-        padding-right: .5rem;
+        padding-left: -.5rem;
+        padding-right: -.5rem;
```
Comment (jordangehman) [inline dist/aw-grid.css:554]: What are these changes for? I can't see the context in GitHub.

File: src/_functions.scss [hunk 2]
```diff
@@ -1,7 +1,26 @@
...
 }
 
+/// Verifies if the value is a percentage or another kind of length in order
+/// to remove or return a percentage value to be used as a multiplier unit
+///
+/// @access private
+///
+/// @param {Length} $value - Length unit to be checked
+///
+/// @return {Number | Length}
```
Comment (jordangehman) [inline src/_functions.scss:23]: Why three forward slashes? Two wasn't good enough? ;)

**Review by jordangehman — COMMENTED** [review]
Does the SASS Docs favicon.png need to be included?

File: src/_functions.scss (hunk 2 above)
Comment (jordangehman) [reply src/_functions.scss:23]: Ah, I see it looks like that's what SASSDOC requires.

Comment (davegomez) [reply src/_functions.scss:23]: SassDoc standard to generate the documentation files.

File: dist/aw-grid.css (hunk 1 above)
Comment (davegomez) [reply dist/aw-grid.css:554]: The `padding-left` and `padding-right` values should be negative as described in [this PR](https://github.com/kristoferjoseph/flexboxgrid/commit/f36e4d364b87f0218226d66324d912e805a6ceca) from Flexbox Grid fixing the [Issue #75](https://github.com/kristoferjoseph/flexboxgrid/pull/75).

**Pull request merged**